packages/ui/src/forum/components/PostList/PostListItem.tsx

```diff
@@ -65,6 +67,20 @@ export const PostListItem = ({
     !!ref.current && insertRef && insertRef(ref)
   }, [ref.current])
 
+  useEffect(() => {
+    if (ref.current && isSelected) {
+      ref.current.scrollIntoView({ behavior: 'smooth', inline: 'start' })
+      const timeout = setTimeout(() => {
+        clearSelection?.()
+        // timeout to clear params after glow animation
+      }, 1500)
+
+      return () => {
+        clearTimeout(timeout)
+      }
+    }
+  }, [isSelected, ref.current])
```

Comment:
Yeah, so the diff between stack and us is that we allow users to navigate to the replied post. So if we leave the post uncleared when someone navigates to one post, scrolls again, and tries to navigate to the same post for the second time it will fail. 

Two most simple solutions: 
- Don't bother about it not working for the second time
- Fix the page problem and leave it as it is (removing `navigation.post` from offset deps should do it)

Reply:
Between the 2 I would prefer the solution "Don't bother about it not working for the second time" because IMO picking a post is a navigation action that should be in the history but I think this might work:
```suggestion
  useEffect(() => {
    if (ref.current && isSelected) {
      ref.current.scrollIntoView({ behavior: 'smooth', inline: 'start' })
    }
  }, [location, ref.current])
```
With `const location = useLocation()`